src/components/Routes.jsx

```diff
@@ -0,0 +1,87 @@
+import { Route, Switch } from 'react-router-dom';
+import UsersWithContext from './ContextHOCs/UsersWithContext';
+import Tasks from '../pages/Tasks';
+import UsersTasksWithContext from './ContextHOCs/UsersTasksWithContext';
+import UsersTracks from '../pages/UsersTracks';
+import UsersProgress from '../pages/UsersProgress';
+import PageNotFound from '../pages/PageNotFound';
+import settingsIcon from '../icons/settings.svg';
+import usersIcon from '../icons/user.svg';
+import tasksIcon from '../icons/tasks.svg';
+import MenuItem from './Aside/Menu/MenuItem/MenuItem';
+import SettingsWithContext from './ContextHOCs/SettingsWithContext';
+
+const paths = [
+  { isExact: true, path: '/main/users', component: UsersWithContext },
+  { isExact: true, path: '/main/tasks', component: Tasks },
+  { isExact: true, path: '/main/users/:userId/tasks', component: UsersTasksWithContext },
+  { isExact: true, path: '/main/users/:userId/tasks/:taskId/track', component: UsersTracks },
+  { isExact: true, path: '/main/users/:userId/progress', component: UsersProgress },
+  { isExact: true, path: '/main/settings', component: SettingsWithContext },
+  { isExact: false, path: '', component: PageNotFound },
+];
+
+const adminAndMentorPermissions = [0, 1, 2, 3, 4, 5, 6];
+const userPermissions = [2, 3, 5, 6];
+
+export const getRoleDependedRoutes = (loggedUser) => {
+  let permissionsList = null;
```

Comment:
you can set it to `userPermissions` by default for example

Reply:
It was refactored according your previous task:
```suggestion
export const getRoleDependedRoutes = ({ role }) => {
  const rolePermissions = paths.filter((pathObj) => pathObj.permissions.includes(role));
  const routePack = rolePermissions.map((path) => (
    <Route key={Date.now().toString()} exact={path.isExact} path={path.path} component={path.component} />
  ));

  return <Switch>{routePack}</Switch>;
};
```